### DISCUSSION TITLE
Exporting dynamic recipient field

### DISCUSSION THREAD

**Discussion by rexterkate-adwise** [solspace/craft-freeform#407] [Questions & Answers]
When exporting our submissions the dynamic recipient field is exported as an empty column. Is it possible to export 'dynamic recipient' data?

**Comment by kjmartens** [comment]
Hi @rexterkate-adwise, can you clarify which version of Freeform you're currently using? I did a quick test in Freeform and don't appear to be experiencing this issue, but maybe you're using an older version or doing something differently. 🙂
